draft-ietf-httpbis-priority.md

```diff
@@ -784,12 +783,13 @@ request ordering to influence response order.
 
 Incremental responses of the same urgency SHOULD be served by sharing bandwidth
 among them. The message content of incremental responses is used as parts, or
-chunks, are received. A client might benefit more from receiving a portion of
-all these resources rather than the entirety of a single resource. How large a
-portion of the resource is needed to be useful in improving performance varies.
-Some resource types place critical elements early; others can use information
-progressively. This scheme provides no explicit mandate about how a server
-should use size, type, or any other input to decide how to prioritize.
+chunks, as they are received. A client might benefit more from receiving a
```

Comment:
I don't follow the proposed sentence

> The message content of incremental responses is used as parts, or chunks, as they are received.

The main issue is the double occurence of "as".

My intention was to say "used in parts, as they are received". But it seems you have a different intent here. Maybe something like "The message content of incremental responses can be used as soon as parts, or chunks, of the content is received"?

Reply:
Thanks. I had problem parsing the new text, but rereading it, I think it works. Reverted the proposed change in 84c4b9a.